doc/rst/technotes/libraries.rst

```diff
@@ -551,6 +558,149 @@ include the chapel runtime and standard modules for use in a non-Chapel program
 and when the library is linked to a Chapel program this leads to multiple
 definitions of these functions.
 
+Using Your Library in Multilocale Settings
+==========================================
+
+Prerequisites
+-------------
+
+Chapel supports ``--library`` when ``CHPL_COMM = gasnet``.  We intend to support
```

Comment:
Doesn't Chapel support `--library` when `CHPL_COMM = none` and  `CHPL_LAUNCHER=none`? Being more specific might reduce confusion.

Reply:
Yup!  Would "Chapel also supports" be enough to help?

Reply:
That would work.